esmvaltool/preprocessor/_mask.py

```diff
@@ -129,6 +136,29 @@ def mask_landsea(cube, fx_files, mask_out):
     return cube
 
 
+def mask_landseaice(cube, fx_files, mask_out):
+    """Apply a land/ice mask"""
```

Comment:
Add [numpy style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) documentation to this function as it is a publicly visible preprocessor function.